hack/cloud-image-uploader/aws.go

```diff
@@ -139,6 +140,42 @@ func (au *AWSUploader) registerAMI(ctx context.Context, region string, svc *ec2.
 		}
 	}()
 
+	denyInsecurePolicy := map[string]interface{}{
```

Comment:
let's just define this as a var and just do `fmt.Sprintf` and use that as string

Reply:
Done, but imo that looks way worse :sweat_smile:

Reply:
oh  I meant define as a `var ()` block at the top, but nothing critical here

Reply:
OK, I redid that then :)